docs/content/en/usage.md

```diff
@@ -104,6 +104,47 @@ You can also define your Strapi entities to add shortcuts to the previous method
 ```js
 await this.$strapi.$products.find()
 ```
+## Usage with the Nuxt Composition Api
+
+First you need to install the [Nuxt Composition API](https://composition-api.nuxtjs.org/) and add it to your project
+
+In order to use the Composition API you must be on Nuxt v2.12 or above because we will be using the useFetch Hook instead of useAsync due to the fact that useAsync [won't re call the information after inital load](https://composition-api.nuxtjs.org/helpers/useAsync)
+
+Your setup method will look like this
+
+```
+import { useFetch, ref } from '@nuxtjs/composition-api'
+export default{
+  setup() {
+          const homepage = ref()
+          useFetch(async ({ $strapi }) => {
+              homepage.value = await $strapi.find('home')
+          })
+          return { homepage }
+      },
+}
+```
+> :warning: Remember that the JSON data is case sensitive
+
+If you want to use dynamic route matching you can pass in the url parameters by doing this
+
+```
+import { ref, useFetch, useRoute } from '@nuxtjs/composition-api'
+export default {
+    name: '_id',
+    setup() {
+        const article = ref()
+        const route = useRoute()
+        useFetch(async ({ $strapi }) => {
+            article.value = await $strapi.find('articles', {
+                slug: route.value.params.id,
+            })
+        })
```

Comment:
Looks good to me! Another option would be to deconstruct `$route` from the component instance as with `$strapi`, e.g.:
```suggestion
        useFetch(async ({ $strapi, $route }) => {
            article.value = await $strapi.find('articles', {
                slug: $route.params.id,
            })
        })
```

Reply:
Will $route work within the setup() method? I know nuxt is in a weird stage where its not vue 3 but has the composition api. If it were vue 3 $route would not work but this may work.

Reply:
@Cpaczek Yes it should.